plugins/modules/monitoring/datadog/datadog_monitor.py

```diff
@@ -106,9 +106,17 @@
     thresholds:
         description:
           - A dictionary of thresholds by status.
-          - Only available for service checks and metric alerts.
+          - Only available for service checks, metric alerts, and query alerts.
           - Because each of them can have multiple thresholds, we do not define them directly in the query.
         default: {'ok': 1, 'critical': 1, 'warning': 1}
+    threshold_windows:
+        description:
+          - A dictionary of threshold windows by status
+          - These options only apply to anomaly monitors which are of the "query alert" type; Use elsewhere will be ignored.
+          - Acceptable values per the DataDog API are: last_#m (1, 5, 10, 15, or 30), last_#h (1, 2, or 4), or last_#d (1 or 2)
```

Comment:
What you are doing here is defining a list item which contains a dictionary with one key `'Acceptable values per the DataDog API are'` and value `'last_#m (1, 5, 10, 15, or 30), last_#h (1, 2, or 4), or last_#d (1 or 2)'`. That's why it is best to use quotes when dealing with complex strings in YAML.

(That's also why the next two lines work well - because they are quoted.)

Reply:
Wow, I didn't realize you could put spaces in a key. Assumed that would make it a a string. Nice. Thanks for pointing that out!

Reply:
YAML (and Python) dict keys can be a lot of different things ;)

In YAML, it's usually a good idea to quote every string that has a `:` in it, just to be safe ;)